packages/core/src/local-admin-api.ts

```diff
@@ -7,27 +7,39 @@ import { LocalIndexApi } from './indexing/local-index-api.js'
  */
 export class LocalAdminApi implements AdminApi {
   constructor(
+    private readonly adminDids: Array<string> | undefined,
     private readonly indexApi: LocalIndexApi,
     private readonly logger: DiagnosticsLogger
   ) {}
 
-  async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
+  private verifyActingDid(actingDid: string) {
+    if (!this.adminDids || !this.adminDids.some( adminDid => actingDid.startsWith(adminDid) )) {
+      this.logger.log(LogStyle.warn, `Unauthorized access attempt to Admin Api from did: ${actingDid}`)
+      throw Error('Unauthorized access')
```

Comment:
```suggestion
      throw Error(`Unauthorized access: DID '${actingDid}' does not have admin access permission to this node`)
```

Reply:
users might not even know where the composedb gets its DID from, including the DID being used in the error message may help node operators figure out what's going on when they have a misconfiguration